spec/index.md

```diff
@@ -4994,6 +5059,8 @@ S with
     reserved_balance_limits[Canister_id] = New_reserved_balance_limit
     certified_data[Canister_id] = ""
     canister_history[Canister_id] = New_canister_history
+    canister_log_visibility[Canister_id] = Public
```

Comment:
Is Public gonna be the default log visibility?

Reply:
I suppose it's safer to go with controllers by default.